api/DataTransfer.json

```diff
@@ -0,0 +1,556 @@
+{
+  "api": {
+    "DataTransfer": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DataTransfer",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "3.5",
+            "notes": "As of Firefox 52, the <a href='https://developer.mozilla.org/docs/Web/API/DataTransfer/types'><code>DataTransfer.types</code></a> property returns a frozen array of <a href='https://developer.mozilla.org/docs/Web/API/DOMString'><code>DOMString</code></a>s as per spec, rather than a <a href='https://developer.mozilla.org/docs/Web/API/DOMStringList'><code>DOMStringList</code></a>."
+          },
+          "firefox_android": {
+            "version_added": "52"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "12"
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": "3.1"
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "Constructor": {
```

Comment:
This should be called DataTransfer as well, but with a description that specifies it's a constructor.

Reply:
Thanks, I have fixed this in d281f23.